py-polars/polars/dataframe/frame.py

```diff
@@ -409,22 +408,6 @@ def _from_pydf(cls, py_df: PyDataFrame) -> Self:
         df._df = py_df
         return df
 
-    @classmethod
-    def _from_dicts(
```

Comment:
Not 100% I'm willing to part with this code yet. I want to give some thought on these private classmethods - maybe we should get rid of all of them. But let's keep them around until I've decided.

Reply:
It is superseded by `sequence_to_pydf`,  but I had to triple check myself to ensure it was all good, as it is a bit odd this code is not in use given all the other private methods on DataFrame. Reverted.